### DISCUSSION TITLE
Make use of empty space on big screens

### DISCUSSION THREAD

**Discussion by IamShubhamGupto** [alshedivat/al-folio#1508] [Ideas]
The current theme center aligns the content and leaves a lot of unused space on the left and right side - this is clearly visible for large screens.

Suggestion is add the ability to have some sort of animations or additional text making use of this space

other suggestions to properly utilize screen space is welcome

my current setup - https://iamshubhamgupto.github.io

**Comment by george-gca** [comment]
You could also simply increase the max width of the site. This can be done in the `_config.yml` file, increasing the current `max_width: 800px`. Adding animations would make the site slower.
